mldsa/ntt.c

```diff
@@ -151,31 +189,25 @@ void ntt(int32_t a[MLDSA_N])
  *              MLDSA_Q in absolute value. Output coefficient are smaller than
  *MLDSA_Q in absolute value.
```

Comment:
Fix indenting of this comment please.

Reply:
done (this was auto generated when we changed variable names with the prefix and ran linting)